modules/apps/frontend-taglib/frontend-taglib-clay/src/main/resources/META-INF/resources/data_set_display/management_bar/components/NavBar.js

```diff
@@ -40,7 +40,9 @@ function NavBar({creationMenu, showSearch, views}) {
 					</div>
 				)}
 				<div className="navbar-form navbar-form-autofit navbar-overlay navbar-overlay-sm-down pl-0">
-					{views?.length > 1 && <ActiveViewSelector views={views} />}
+					{views?.length > 1 ? (
```

Comment:
Why do we need the ternary here?

Reply:
In case the array is empty it should display nothing. At the moment it displays "0"

Reply:
I mean, can't we do `{views?.length > 1 && ( <ActiveViewSelector views={views} /> )`?

Reply:
I was probably drunk when I wrote my previous comment. You're totally right 💯